docs/scos/dev/front-end-development/202307.0/oryx/building-components/component-integration.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "Integrate Oryx Components in any web frameworks"
+description: Oryx Components are build as web components
+last_updated: Sept 20, 2023
+template: concept-topic-template
+---
+
+Oryx components are _framework agnostic_. This means that components can be used in other web frameworks.
+
+Oryx components are build as [web components](https://developer.mozilla.org/en-US/docs/Web/API/Web_components). Web components are a suite of standard web technologies, widely embraced by browser vendors. The purpose of web components is to provide components in isolation, so that they can easily integrate with other web technologies.
+
+Web components can be built with different frameworks or even plain html, css and javascript. Oryx components are implemented with [Lit](https://lit.dev). Lit is a lightweight open source framework from Google that can be used to build highly efficient web components. If you do not want to use Lit, you can use your own framework of choice.
```

Comment:
I think I read this Lit and "if you don't what to use lit" thing in earlier chapters, and it's not so relevant to this one, but just a suggestion